boa/src/syntax/parser/statement/if_stm/mod.rs

```diff
@@ -2,6 +2,7 @@
 mod tests;
 
 use super::Statement;
+
```

Comment:
Why this new line?

Reply:
I imagine rustfmt did that as it is not something I did directly